src/main/java/com/wcc/platform/surrealDB/SurrealDBWebSocketClient.java

```diff
@@ -0,0 +1,56 @@
+package com.wcc.platform.surrealDB;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.util.concurrent.CompletableFuture;
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.WebSocketHttpHeaders;
+import org.springframework.web.socket.WebSocketSession;
+import org.springframework.web.socket.client.standard.StandardWebSocketClient;
+import org.springframework.web.socket.handler.TextWebSocketHandler;
+
+@Component
+public class SurrealDBWebSocketClient extends TextWebSocketHandler {
+
+  private final ObjectMapper objectMapper = new ObjectMapper();
+  private WebSocketSession session;
+
+  public CompletableFuture<WebSocketSession> connect(String uri) throws Exception {
```

Comment:
I am interested to understand more about CompletableFuture class and its usage (also about the Async annotation - when to use it?). @dricazenck @goelsonali maybe during one of the next meetings?